ntoskrnl/io/iomgr/irq.c

```diff
@@ -166,4 +166,67 @@ IoDisconnectInterrupt(PKINTERRUPT InterruptObject)
     ExFreePool(IoInterrupt); // ExFreePoolWithTag(IoInterrupt, TAG_KINTERRUPT);
 }
 
+#if (NTDDI_VERSION >= NTDDI_LONGHORN) || defined(__REACTOS__)
```

Comment:
I don't think this needs to be guarded with ifdefs

Reply:
Just the private function? Or the whole thing?

Reply:
The whole guard is not needed